src/coreclr/src/vm/peimagelayout.h

```diff
@@ -110,7 +110,10 @@ class ConvertedImageLayout: public PEImageLayout
 public:
 #ifndef DACCESS_COMPILE
     ConvertedImageLayout(PEImageLayout* source);
+    virtual ~ConvertedImageLayout();
```

Comment:
Could you make the class `final` to avoid having a virtual destructor?

Reply:
The base class has virtual destructor, so this `virtual` is somewhat redundant and is mostly for readability. It would be a virtual override anyways. It is also possible to not specify anything or specify `override`. I do not have a lot of preference here as long as compilers can handle it and it is familiar to readers. 

I do not see a lot of use of `final` though. The codebase is mostly pre-`C++11`, except when needed to expresses something that was not possible before.